contributing-to-the-network/setup-host-using-staking-gui.md

```diff
@@ -0,0 +1,80 @@
+
+## Requirements
+A couple things before you can use the Staking GUI:
+1. Make sure your device meets the minimum requirements for a host: https://wiki.edge.network/contributing-to-the-network/network-nodes#minimum-specification
+2. Download, Install & Run Docker: https://docs.docker.com/get-docker/
+3. Download, Install & Run Pod's Staking GUI: https://github.com/PodTheCoder/edge_staking_gui/releases
+
+## Visual Guide
+1. (Web) Create a Host stake: https://wallet.xe.network/staking
+- Click create stake
+<img width="984" alt="Create Stake - Copy" src="https://github.com/edge/wiki/assets/113918164/8c87ccb9-c498-4bfc-b042-38ff59e1303f">
+
+- Choose stake type: Host. Enter your password.
+  
+<img width="448" alt="Create Stake2" src="https://github.com/edge/wiki/assets/113918164/c9d6f06f-8ff8-4576-8eac-f69dd534f2bc">
+
+- Click the button to create a stake.
+<img width="442" alt="Create Stake 3" src="https://github.com/edge/wiki/assets/113918164/ef0019f9-1e5b-4b1f-abf6-223d7ca901fd">
+
+
+2. (GUI) Install the latest Edge CLI
+
+<img width="183" alt="2 install Edge CLI" src="https://github.com/edge/wiki/assets/113918164/a94ca24a-548b-452c-b3be-e4cc147ec7ea">
+
+4. (GUI) Assign Device
+
+- Click "Create & Display Device Token"
+
+<img width="244" alt="3 install create   display device token - Copy" src="https://github.com/edge/wiki/assets/113918164/51c7cbf7-056f-4dc8-9d97-b97b783b3e45">
+
+- Copy the device code:
+<img width="582" alt="4 assign device 2 - Copy" src="https://github.com/edge/wiki/assets/113918164/04d21c6f-1a5e-4168-a99e-bf976df8876e">
+
+6. (Web) Assign Device
+
+- Navigate to https://wallet.xe.network/staking & click the "Assign Device" button.
+
+<img width="955" alt="4 assign device - Copy" src="https://github.com/edge/wiki/assets/113918164/304ac622-5496-4e19-a23d-f73303d0b00e">
+
+- Paste the device code:
+
+<img width="426" alt="4 assign device 3 - Copy" src="https://github.com/edge/wiki/assets/113918164/d8ae91ae-5788-4c33-9d25-b02e634a536f">
+
+- Wait 2-5 minutes until your device assignment is confirmed on the blockchain.
+  
+<img width="446" alt="4 assign device 4 - Copy" src="https://github.com/edge/wiki/assets/113918164/4a5ce253-bc1a-4dd4-a78b-c8b56444514b">
+
+7. (GUI) Start your node.
+
+- Click the button.
+
+<img width="219" alt="5 start node - Copy" src="https://github.com/edge/wiki/assets/113918164/d5ed740f-415b-4173-843f-b093bf350036">
+
+- Wait. This can take up to an hour.
+
+8. (GUI) Setup complete!
+
+<img width="597" alt="6 success - Copy" src="https://github.com/edge/wiki/assets/113918164/00864ed2-704f-4454-89e7-995884213b2f">
+
+9. (Docker, Optional) Autostart docker.
+
+- Click the gear icon.
+
+![image](https://github.com/edge/wiki/assets/113918164/60a51390-91d2-40ea-99b7-9cb065f167e7)
+
+- Check "Start Docker Desktop when you log in"
+
+![image](https://github.com/edge/wiki/assets/113918164/bfe437d7-9332-41ae-93b1-7bfd2985afae)
```

Comment:
Some users may be using Docker Engine rather than Docker Desktop, so clarification required here I think: "If you are using Docker Desktop, you may need to enable autostart."

We can probably omit autostart for Docker Engine as it autostarts by default and if not, the user is probably on Linux and knows (or can find out) how to manage autostart.